programs/bpf_loader/src/syscalls.rs

```diff
@@ -317,6 +322,16 @@ pub fn bind_syscall_context_objects<'a>(
         }),
     );
 
+    bind_feature_gated_syscall_context_object!(
+        vm,
+        invoke_context.is_feature_active(&secp256k1_recover_syscall_enabled::id()),
+        Box::new(SyscallSecp256k1Recover {
+            base_cost: bpf_compute_budget.secp256k1_recover_base_cost,
```

Comment:
just "cost", there is not per-byte cost associated with this so no need to distinguish this as a "base" cost

Reply:
Fixed